source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -7,13 +7,26 @@
   {%- with logo|default({})|merge({html: html}) only -%}
     {%- block logo -%}
 
-      {%- set image = image|default({})|merge({src: image.src|default(asset('images/libero-logo.svg')), alt: image.alt|default('Site logo'), class: image.class|default([])|merge(['site-header__image'])}) -%}
+      {%- set image = image|default({})|merge({alt: image.alt|default('Site logo'), class: image.class|default([])|merge(['site-header__image'])}) -%}
 
       {%- if href is defined -%}
         <a href="{{ href }}" class="site-header__logo_link">
       {%- endif -%}
 
-      <img {{- html.attributes(image) }}>
+      {%- if image.src is defined -%}
+
+        <img {{- html.attributes(image) }}>
+
+      {%- else -%}
+
+        {%- set image = image|merge({src: asset('images/libero-logo-small.svg')}) -%}
```

Comment:
Should be able to use `libero-logo.svg#graphic`, but IDs are currently being stripped so doesn't work.

Reply:
Not needed, so removing.